osgp/protocol-adapter-oslp/osgp-adapter-protocol-oslp-elster/src/main/java/org/opensmartgridplatform/adapter/protocol/oslp/elster/infra/messaging/MessageMetadataFactory.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Alliander N.V.
+ */
+
+package org.opensmartgridplatform.adapter.protocol.oslp.elster.infra.messaging;
+
+import org.opensmartgridplatform.adapter.protocol.oslp.elster.device.DeviceResponse;
+import org.opensmartgridplatform.shared.infra.jms.MessageMetadata;
+
+public class MessageMetadataFactory {
+  public static MessageMetadata from(
```

Comment:
Sonar zegt hier: "Add a private constructor to hide the implicit public one." Misschien dat nog even doen. Verder heb ik geen opmerkingen :)